timesketch/lib/analyzers/ntfs_timestomp.py

```diff
@@ -121,16 +121,14 @@ def run(self):
 
         timestomps = 0
         for file_info in file_infos.values():
-            if self.handle_timestomp(file_info):
+            if self.is_suspicious(file_info):
```

Comment:
I know this is not the correct place for this, but since you are making changes, can you change the docstring for the `is_suspicious` function, since it is longer than a single line, and it shouldn't be.

Reply:
Done.